molecule/api.py

```diff
@@ -9,49 +9,47 @@
 LOG = logger.get_logger(__name__)
 
 
-CORE_DRIVERS = [
-    "delegated",
-    "digitalocean",
-    "docker",
-    "ec2",
-    "gce",
-    "hetznercloud",
-    "linode",
-    "lxc",
-    "lxd",
-    "openstack",
-    "podman",
-    "vagrant",
-]
+class UserListMap(UserList):
```

Comment:
```suggestion
class UserListDict(UserList):
```

Bit more Python centric?

Reply:
I will fix it but when I have to touch it next time.